2025-03-08 20:51:03 - bot_logger - ERROR - [base.py:14] - Error creating database tables: Table 'users' is already defined for this MetaData instance.  Specify 'extend_existing=True' to redefine options and columns on an existing Table object.
